.github/workflows/docker.yml

```diff
@@ -4,9 +4,104 @@ on: [push]
 jobs:
   docker-build:
     runs-on: ubuntu-latest
+    services:
+      registry:
+        image: registry:2
+        ports:
+          - '5000:5000'
+    env:
+      DOCKER_BUILDKIT: 1
+      DOCKER_CLI_EXPERIMENTAL: enabled
     steps:
+    - name: enable experimental mode
+      run: |
+        mkdir ~/.docker
+        echo '{"experimental": "enabled"}' > ~/.docker/config.json
+    - name: Install BuildX
+      run: |
+        mkdir -p ~/.docker/cli-plugins
+        curl -SL -o ~/.docker/cli-plugins/docker-buildx https://github.com/docker/buildx/releases/download/v0.3.1/buildx-v0.3.1.linux-amd64
+        chmod 755 ~/.docker/cli-plugins/docker-buildx
+    - name: Create BuildX Builder
+      run: |
+        docker buildx create \
+          --name builder \
+          --platform linux/amd64,linux/arm64,linux/arm/v6,linux/arm/v7,linux/ppc64le,linux/s390x,linux/386 \
+          --use \
+          --driver docker-container --driver-opt image=moby/buildkit:buildx-stable-1,network=host
+        docker buildx ls
+        docker ps
+        docker images
+    - run: docker info && docker version
     - uses: actions/checkout@v1
-    - name: Make Docker images
-      run: make docker-images
+    - name: determine if this is a tag
+      run: |
+        if (git describe --abbrev=0 --exact-match &>/dev/null); then
```

Comment:
you don't need subshell here:

```
        if git describe --abbrev=0 --exact-match &>/dev/null; then
```

Reply:
thanks @glensc - this is a holdover from many copy-pastes ago... I'll happily merge a PR if you want to make the change 😉

Reply:
no worries. you can leave as is as well. totally up to you.